pkg/monitors/filesystems/partitions_windows.go

```diff
@@ -0,0 +1,123 @@
+package filesystems
+
+import (
+	"strings"
+	"syscall"
+
+	gopsutil "github.com/shirou/gopsutil/disk"
+	"golang.org/x/sys/windows"
+)
+
+func (m *Monitor) getPartitions(all bool) ([]gopsutil.PartitionStat, error) {
+	drivePartitionStats, err := gopsutil.Partitions(all)
+	if err != nil {
+		return drivePartitionStats, err
+	}
+
+	allMountPoints := getAllMountPoints()
+	folderMountPoints := make([]string, 0)
+
+	found := false
+	for _, mountPoint := range allMountPoints {
+		for _, drivePartitionStat := range drivePartitionStats {
+			if mountPoint == drivePartitionStat.Mountpoint {
+				found = true
+				break
+			}
+		}
+		if found {
+			found = false
+			continue
+		}
+		folderMountPoints = append(folderMountPoints, mountPoint)
+	}
```

Comment:
```suggestion
	for _, mountPoint := range allMountPoints {
		found := false
		for _, drivePartitionStat := range drivePartitionStats {
			if mountPoint == drivePartitionStat.Mountpoint {
				found = true
				break
			}
		}
		if !found {
			folderMountPoints = append(folderMountPoints, mountPoint)
		}
	}
```

imo this is clearer, but I'm not sure I understand the intended flow overall.  Is `allMountPoints` a superset of `drivePartitionStats(true)` or is it missing some of what's reported there?  Have you reimplemented or fixed a ~bug in `gopustil.Partitions(true)`?

Reply:
Yes, `allMountPoints` is a superset of the set of drivePartitionStat.Mountpoint. No, to the second question.